store/store.go

```diff
@@ -314,6 +317,17 @@ func init() {
 	}
 	defaultConfig.DetailFields = jsonutil.StructFields((*snapDetails)(nil), "snap_yaml_raw")
 	defaultConfig.InfoFields = jsonutil.StructFields((*storeSnap)(nil), "snap-yaml")
+	defaultConfig.SearchFields = append(jsonutil.StructFields((*storeSnap)(nil),
+		"architectures", "created-at", "epoch", "name", "snap-id", "snap-yaml"),
+		"channel")
+}
+
+type searchV2Results struct {
+	Results   []*storeSearchResult `json:"results"`
+	ErrorList []struct {
```

Comment:
we don't do anything with this? could you find out if there's any errors that is worth specializing for? or is all things like bad request? maybe is still worth  to do something with this for better debugging?

Reply:
Handled now. As for potentially specializing any of those errors I'm waiting for an error list from @tonysimpson